PC/winsound.c

```diff
@@ -95,13 +95,15 @@ winsound_PlaySound_impl(PyObject *module, PyObject *sound, int flags)
         }
         wsound = (wchar_t *)view.buf;
     } else {
-        if (!PyUnicode_Check(sound)) {
+        PyObject *obj;
+        if (PyBytes_Check(sound) || (obj = PyOS_FSPath(sound)) == NULL) {
             PyErr_Format(PyExc_TypeError,
-                         "'sound' must be str or None, not '%s'",
+                         "'sound' must be str, pathlib.Path, or None; not '%s'",
```

Comment:
I think this should say `os.PathLike` rather than `pathlib.Path`. Per [the docs for `PyOS_FSPath()`](https://docs.python.org/3/c-api/sys.html#c.PyOS_FSPath):

> If the object implements the os.PathLike interface, then __fspath__() is returned as long as it is a str or bytes object.

Also affects the `NEWS` entry.

Reply:
Agree. If you change this, I can merge the PR.